### PR TITLE
General test remove user

### PR DESCRIPTION
**Pull request by ilonalink**

### CODE REVIEW

File: cypress/page-objects/teamPage.ts [hunk 1]
```diff
@@ -3,6 +3,23 @@ import { Constans } from "./../infrastructure/consts"
...
+        cy.selectFromMultiSelectDropDown('class','_med','Active');
+        let activeUsersTable = cy.get('.rt-table').should('exist');
+        cy.get('.rt-table .rt-tbody .rt-td').should('have.length.greaterThan',0);
+        return activeUsersTable;
+    }
+
+    public get getDisabledUsers() {
+        this.cleanTeamFilter();
+        cy.selectFromMultiSelectDropDown('class','_med','Disabled');
+        cy.get(':nth-child(6) > .rt-resizable-header-content').click().click();
```
Comment (ShannyWorkiz) [inline cypress/page-objects/teamPage.ts:18]: @ilonalink Why do we have 2 double clicks here?

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -112,6 +113,31 @@ Cypress.Commands.add('waitForToasterToDisappear', () => {
     cy.get('.toastr', {timeout: 10000}).should('not.exist');
 });
 
+Cypress.Commands.add('selectFromMultiSelectDropDown', (by: string, element: string, valueToChoose: string) => {
```
Comment (ShannyWorkiz) [inline cypress/support/commands.ts:116]: @ilonalink What is the difference between 'selectFromDropDown' to 'MultiSelectDropDown'?

File: cypress/tests/core2/general.spec.ts [hunk 3]
```diff
@@ -33,4 +34,29 @@ describe('general tests',() => {
         userRole = userPage.getUserRole;
         userRole.should('equal',` ${UserRoles.Dispatch}`);
     });
+
+    it('After remove user it will not appear on active users screen',() => {
+        let loginPage = new LogInPage;
+        loginPage.logInWithAccount2();
```
Comment (ShannyWorkiz) [inline cypress/tests/core2/general.spec.ts:40]: @ilonalink If we repeat it for every test we can put it in before each, no?

File: cypress/page-objects/teamPage.ts (hunk 1 above)
Comment (ilonalink) [reply cypress/page-objects/teamPage.ts:18]: to make column Created from the newest date to old, ordering table

File: cypress/support/commands.ts (hunk 2 above)
Comment (ilonalink) [reply cypress/support/commands.ts:116]: different elements to search, selectFromDropDown doesn't work here :/

File: cypress/tests/core2/general.spec.ts (hunk 3 above)
Comment (ilonalink) [reply cypress/tests/core2/general.spec.ts:40]: yes, will do

File: cypress/page-objects/teamPage.ts (hunk 1 above)
Comment (ShannyWorkiz) [reply cypress/page-objects/teamPage.ts:18]: @ilonalink Maybe we should create a general ordering function, I've wrote one too.
Let's create a task for this :)
This is Ok for now

File: cypress/support/commands.ts (hunk 2 above)
Comment (ShannyWorkiz) [reply cypress/support/commands.ts:116]: Got it, thanks!

File: cypress/page-objects/teamPage.ts (hunk 1 above)
Comment (ilonalink) [reply cypress/page-objects/teamPage.ts:18]: alright ;)

**Review by ShannyWorkiz — APPROVED** [review]

**Pull request merged**